src/components/coin-row/BalanceCoinRow.js

```diff
@@ -24,7 +24,7 @@ const formatPercentageString = percentString =>
   percentString ? percentString.split('-').join('- ') : '-';
 
 const BalanceCoinRowCoinCheckButton = styled(CoinCheckButton).attrs({
-  isAbsolute: true,
+  left: 9.5,
```

Comment:
This number seems to be coupled to the `19px` value used internally in `CoinCheckButton` and the layout is broken if any other value is used. Should the `left` prop be removed and the value be inlined within `CoinCheckButton`?

Reply:
Hm, I don't know. I have the opinion that we should require this component soon so I think this will be good to make this kind of "safe" change that will potentially not impact anything around.